common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -312,55 +341,16 @@ def gate_sequence_if_it_is_a_timed_exam_and_contains_content_type_gated_problems
         We are displaying the gating fragment within the sequence, as is done for gating for prereqs,
         rather than content type gating the entire sequence because that would remove the next/previous navigation.
 
-        When gated_sequence_fragment is not set to None, the sequence will be gated.
+        When gated_sequence_paywall is not set to None, the sequence will be gated.
 
         This functionality still needs to be replicated in the frontend-app-learning courseware MFE
         The ticket to track this is https://openedx.atlassian.net/browse/REV-1220
         Note that this will break compatability with using sequences outside of edx-platform
         but we are ok with this for now
         """
-        if not self.is_time_limited:
```

Comment:
this method i previously worked on for a different ticket was checking if a sequence contained any content type gating content, so most of that code does the same thing that I needed for this ticket. 
I tried to make it DRY so most of this method is deleted